### ISSUE TITLE
What's happening with this repository once OpenNI is offline?

### ISSUE THREAD

**Issue by schwittlick** [OpenNI/OpenNI2#75]
At the end of april OpenNI is not going to be available anymore. What does this mean for people who are still using OpenNI in software- is it going to be prohibited from then on? What's happenin with this repository- is it going to be maintained by OpenNI? Is it legal to copy it to another repository and continue developing it in pure open source style?

Not sure this is the right spot to ask this.

Yours,
Marcel

**Comment by tomoto** [comment]
Hi Marcel,

As to my best knowledge, it should be totally legal for anybody ("volunteers") to copy (fork/clone) it to another repository and continue developing it in pure open source style. Apache License 2.0 under which the current OpenNI is published should allow you to do that.

There are some discussions related to this in the forum (sadly, these links will be available only for a few weeks...). If someone or some group says "I volunteer", I think it will save some people in the world.
http://community.openni.org/openni/topics/fhi0e7pyri05a
http://community.openni.org/openni/topics/new_openni_forum_on_google_groups
http://community.openni.org/openni/topics/can_i_redistribute_openni

Hope it helps,
Tomoto

**Comment by ntx-80** [comment]
Hello Marcel & Tomoto,

At Occipital, when we launched the Structure Sensor, we made a promise to support OpenNI. Since OpenNI.org has been shut down, we’ve created an online resource with OpenNI 2 binaries and documentation. We also made a public fork of the OpenNI 2 repo, which we’ve opened up to the community for pull requests, wiki contributions, and issue tracking. 

Like you folks, we’ve spent a lot of time creating devices and projects that use OpenNI. Let’s work together to keep it available and compatible with the latest operating systems.

Check it out here: http://structure.io/openni

## 

NT

**Comment by tomoto** [comment]
Hi Nicolas, that is great news! I'm sure many people (including myself) will appreciate your effort!

**Comment by CGAnderson** [comment]
Yes, definitely there are lots of people who appreciate your efforts!
Big Gorgeous Mondo Thank You!

On Wed, Apr 23, 2014 at 12:29 PM, Tomoto Shimizu Washio <
notifications@github.com> wrote:

> Hi Nicolas, that is great news! I'm sure many people (including myself)
> will appreciate your effort!
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/OpenNI/OpenNI2/issues/75#issuecomment-41204072
> .

## 

_C.G. Anderson_
_I fell into the upholstery machine the other day, but don't worry, now I'm
completely recovered!_

**Comment by atuleu** [comment]
Hi,

It seems a nice effort. I will watch more closely the future of this repo and certainly continue to provide debian package for openni2 ( I kind of stopped since the primesense was bought). Please see atuleu/debian-openni2@68f45edde31671e60bb651cba51c8964442709ac

**Comment by ntx-80** [comment]
Thanks for the kind words, everyone!

Alexandre, thanks a lot for updating your debian package! Now I'm curious:
is this package part of mainstream Debian? I've briefly looked, but only
found OpenNi 1.5 in Debian repositories. Are there any plans to get OpenNI
2 part of the default distributions?

**Comment by hcw70** [comment]
Hi all. Actually i did some efforts to integrate this into debian / ubuntu, @jspricke has a ppa and works also with some debian guys for tihs...
see https://github.com/hcw70/OpenNI2

**Comment by hcw70** [comment]
Also note that i have successfully running OpenNi2 on RaspberryPi with this...
